src/modules/species/species.service.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Injectable,
+  NotFoundException,
+  UnauthorizedException,
+} from "@nestjs/common";
+import { PrismaService } from "src/config/prisma/prisma.service";
+import { CreateSpeciesDto } from "./dto";
+
+@Injectable()
+export class SpeciesService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async create(CreateSpeciesDto: CreateSpeciesDto) {
```

Comment:
Pode haver um conflito aqui por conta do nome:

```suggestion
  async create(createSpeciesDto: CreateSpeciesDto) {
```